target/linux/ramips/image/mt7621.mk

```diff
@@ -719,6 +723,25 @@ define Device/lenovo_newifi-d1
 endef
 TARGET_DEVICES += lenovo_newifi-d1
 
+define Device/linksys_e5600
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 26624k
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := E5600
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e kmod-mt7663-firmware-ap \
+  uboot-envtools
+  UBINIZE_OPTS := -E 5
+  IMAGES := sysupgrade.bin factory.bin
```

Comment:
```sysupgrade.bin``` is already defined in [IMAGES](https://github.com/openwrt/openwrt/blob/ffeb37047e85a5efd96890db12710e9d60b4b76a/target/linux/ramips/image/Makefile#L166) in image/Makefile, this can be replaced to ```IMAGES += factory.bin```

Reply:
I tried to remove sysupgrade.bin from the list above and sysupgrade bin stopped getting created in the output folder. So, it needs to be kept here. Let me know if I am missing something.

Reply:
Did you replace to ```+=``` from ```:=```?